torch/ao/quantization/_pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -40,6 +42,25 @@ def supported_quantized_operators() -> Dict[str, List[OperatorPatternType]]:
         "conv2d": [
             [torch.nn.Conv2d],
             [F.conv2d],
+            # Conv ReLU
+            [torch.nn.Conv2d, torch.nn.ReLU],
+            [torch.nn.Conv2d, F.relu],
+            [F.conv2d, torch.nn.ReLU],
+            [F.conv2d, F.relu],
```

Comment:
Is it possible to use equivalent mapping to avoid duplication here?

Reply:
I simplify it to use `itertools.product`. It can reduce much duplicated code.